conu/utils/probes.py

```diff
@@ -89,11 +91,15 @@ def _wrapper(self, q, start):
     def _run(self):
         start = time.time()
         fnc_queue = Queue()
+        logger.debug("starting probe")
         p = Process(target=self._wrapper, args=(fnc_queue, start))
         p.start()
+        logger.debug("first process started")
```

Comment:
Should finish with '.'

Reply:
not really, it's not a sentence -- all the logs start lower case and don't end with a dot

Reply:
I actually hate logs which are sentences, from my PoV, it looks horrible -- exceptions should be sentences, since that's what users may see, but logs are for engineers